api/src/test/java/com/epam/pipeline/manager/pipeline/FolderTemplateManagerTest.java

```diff
@@ -75,70 +62,27 @@
     private static final String CHILD_TEMPLATE_FOLDER_NAME_1 = "test-1";
     private static final String DATASTORAGE_NAME_1 = "ds-1";
 
-    @Autowired
+    @SpyBean
     private FolderTemplateManager folderTemplateManager;
     @Autowired
     private FolderManager folderManager;
     @Autowired
     private MetadataManager metadataManager;
-    @Autowired
+    @MockBean
     private DataStorageManager dataStorageManager;
-    @SpyBean
-    private S3StorageProvider storageProviderManager;
-    @Autowired
-    private AclTestDao aclTestDao;
-    @Autowired
+    @MockBean
     private GrantPermissionManager permissionManager;
-    @Autowired
-    private CloudRegionDao cloudRegionDao;
-
-    private AwsRegion awsRegion;
-
-
-    @Before
-    public void setUp() {
-        doReturn(new MockS3Helper()).when(storageProviderManager).getS3Helper(any(S3bucketDataStorage.class));
-
-        awsRegion = new AwsRegion();
-        awsRegion.setName("US");
-        awsRegion.setRegionCode("us-east-1");
-        awsRegion.setDefault(true);
-        cloudRegionDao.create(awsRegion);
+    @MockBean
+    private FolderCrudManager crudManager;
```

Comment:
Similar thing for `FolderManager`, `MetadataManager`, `DataStorageManager` and `FolderCrudManager`. They are already defined as mock beans in `AclTestBeans`. Therefore I would suggest to use `Autowired` annotation for all these classes in `FolderTemplateManagerTest` but rename the fields to `mockFolderManager` and etc.

Additionally could you please define `FolderTemplateManager` bean (not mock) in `AclTestBeans` in order to be consistent.

Reply:
Fixed